packages/mdx/lib/plugin/recma-document.js

```diff
@@ -39,7 +39,6 @@
  *   JSX runtime to use.
  */
 
-import {URL} from 'url'
```

Comment:
There are likely other `URL` imports in other files.
So only changing this line, probably hides one error, but probably shows more errors.
Could you also change the other ones?